docs/pages/build-reference/infrastructure.md

```diff
@@ -146,19 +146,20 @@ When selecting an image for the build you can use the full name provided below o
   - Every build gets its own fresh macOS VM
   - Hardware: Intel(R) Core(TM) i7-8700B CPU (6 cores/12 threads), 64 GB RAM
   - Build resource limits: 3 cores, 12 GB RAM
-- npm cache. [Learn more](caching/#javascript-dependencies)
+- NPM cache. [Learn more](caching/#javascript-dependencies)
+- CocoaPods cache. [Learn more](caching/#ios-dependencies)
 - `~/.npmrc`
 
   ```ini
-  registry=http://10.254.24.8:4873
```

Comment:
Have we changed that?

Reply:
This was staging URL, I changed it to production